pandas/tests/arrays/sparse/test_array.py

```diff
@@ -1255,7 +1255,7 @@ def test_to_coo(
         A, rows, cols = ss.sparse.to_coo(
             row_levels=(0, 1), column_levels=(2, 3), sort_labels=sort_labels
         )
-        assert isinstance(A, scipy.sparse.coo_matrix)
+        assert isinstance(A, scipy.sparse.coo.coo_matrix)
```

Comment:
This should be kept I think? Same one above?

Reply:
Or did they change that back too?

Reply:
Yeah agree this should be kept as is